src/ostree/ot-main.c

```diff
@@ -354,7 +354,7 @@ ostree_option_context_parse (GOptionContext *context,
        * possible to parse */
       g_auto(GStrv) features = g_strsplit (OSTREE_FEATURES, " ", -1);
       g_print ("%s:\n", PACKAGE_NAME);
-      g_print (" Version: %s\n", PACKAGE_VERSION);
+      g_print (" Version: v%s\n", PACKAGE_VERSION);
```

Comment:
How about making this `g_print (" Version: '%s'\n", PACKAGE_VERSION);` instead?

Reply:
Done! :arrow_down: